static/locales/ja.json

```
@@ -426,6 +486,48 @@
+    "configSaved": "設定の保存に成功しました",
+    "saveConfigFailed": "設定の保存に失敗しました: {{error}}",
+    "saveConfigError": "設定保存エラー: {{error}}",
+    "loadingConfig": "設定とモデルファイルを読み込み中...",
+    "configLoaded": "設定は読み込まれましたが、モデルファイルの取得に失敗しました",
+    "loadConfigFailed": "設定の読み込みに失敗しました: {{error}}",
+    "loadConfigError": "設定読み込みエラー: {{error}}",
+    "loadEmotionFailed": "感情マッピング設定の読み込みに失敗しました: {{error}}",
+    "confirmReset": "現在の設定をリセットしてもよろしいですか？すべてのモーションと表情ファイルがクリアされます。",
+    "configReset": "設定がリセットされました",
+    "autoDetected": "{{count}}個のLive2Dモデルを自動検出しました",
+    "autoCreatedMapping": "モデル {{model}} の感情マッピング設定を自動作成しました",
```

Biome (2.3.14)

[error] The key loadCharacterDataFailed was already declared.

This is where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)